website/src/utils.js

```diff
@@ -1,5 +1,6 @@
 var nodemailer = require('nodemailer');
 var nconf = require('nconf');
+nconf.file({ file: './config.json' });
```

Comment:
@crookedneighbor give a look to the line I linked above, it uses __dirname which i think refers to the path from which the script is run so the root of the repository I think but I'm not sure